python/ray/_private/usage/usage_lib.py

```diff
@@ -324,6 +328,24 @@ def record_library_usage(library_usage: str):
         _put_library_usage(library_usage)
 
 
+@contextmanager
+def record_usage(tag_key: TagKey, tag_value: str):
+    """A simple contextmanager fo recording extra usage tag
+
+    Args:
+        tag_key: TagKey defined
+        tag_value: string value
+
+    Example:
+        @record_usage(TagKey.MY_KEY, "MY_VAL")
+        def func(a, b):
+            ...
+
+    """
+    record_extra_usage_tag(tag_key, tag_value)
```

Comment:
Can you verify this is recorded? I think the process needs to have internal kv initialized

Reply:
How could I do that? I followed this basically https://docs.google.com/document/d/1XSWzx9ZatnlttsurCXzvV4vVsb2g1KxJ2T4zpf33bI8/edit not enough for testing?  cc @jjyao

Reply:
I think it is already initialized in the dashboard.

Reply:
is it coming up to the usage_stats.json file?

Reply:
Yep - see the description part.